lua/lspconfig/pyright.lua

```diff
@@ -13,6 +13,7 @@ local root_files = {
   'setup.cfg',
   'requirements.txt',
   '.git',
+  'pyrightconfig.json',
```

Comment:
Not that it matters much because it's a breadth-first search, but I think you should put it before `.git`.

Reply:
thanks, done :)